CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ include_directories(${TRUSTY_ROOT}/trusty/user/base/lib/hwkey/include)
 include_directories(${TRUSTY_ROOT}/trusty/user/base/lib/keymaster/include)
 include_directories(${TRUSTY_ROOT}/trusty/user/base/lib/rng/include)
 include_directories(${TRUSTY_ROOT}/trusty/user/base/lib/storage/include)
+include_directories(${TRUSTY_ROOT}/trusty/user/base/lib/tipc/include)
```

Comment:
it is better to move include directories where they are actually used.

But I guess we can refactor it all at once since other directories are already globally included.